lux/executor/PandasExecutor.py

```diff
@@ -501,7 +501,7 @@ def _is_geographical_attribute(series):
 
     @staticmethod
     def _is_datetime_number(series):
-        if series.dtype == int:
+        if series.dtype == int or "int" in str(series.dtype):
```

Comment:
Why is the second condition needed? And if it is needed, shouldn't there be a cleaner way to search if the Series has an `int` dtype?

Reply:
Yeah so there seems to be some different "int" labels when pulling data from SQL databases. When doing some testing we were getting numpy datatypes like `int64` which was failing this check